### DISCUSSION TITLE
Migrate Library to New Device

### DISCUSSION THREAD

**Discussion by mcfroogle** [PaulWoitaschek/Voice#1637] [General]
I recently got a new phone Samsung S10e and I'm not sure how to move my library across.
I have done it manually before on previous phones but I now have about 200GB of audiobooks (on the microsd card that will be moved to new phone) so might take quite a while to re-mark as read and re-add artwork. Any ideas?

Anyway good work on the app. I use it for about 8 hours a day.

**Comment by PaulWoitaschek** [comment]
Hm, well with version 6.x it might just work out of the box if you backup voice and keep the same folder paths and then add the same audiobook folder. 
Which version are you using?

**Reply by mcfroogle** [reply]
I'm using 6.4.1 how do I "backup" voice is that an android/google drive option? The paths should stay the same I'm planning on just putting the micro sd card in the new phone with no changes. whether the new phone makes up a pathing system or renames stuff.

**Reply by PaulWoitaschek** [reply]
That would be:
https://support.google.com/android/answer/2819582

**Reply by mcfroogle** [reply]
Took the plunged and moved across to the new phone, no luck on the back-up. At least it shouldn't take too long to sort though with the 'mark as read' feature
